### DISCUSSION TITLE
How can I install oh my posh for all centos users?I want all of users use sams theme

### DISCUSSION THREAD

**Discussion by CoderLittleChen** [JanDeDobbeleer/oh-my-posh#3634] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Hi bro,I had installed oh my posh on centos as root user.But when I changed user to general user,the theme had changed.
Should I install oh my posh for every user?
<img width="549" alt="1" src="https://user-images.githubusercontent.com/32502670/227750990-ad93feaf-e01c-4cbd-82e2-3e9a636f235c.PNG">
<img width="550" alt="2" src="https://user-images.githubusercontent.com/32502670/227750991-07798d6e-11ed-4eed-8c7d-df16f4b4466a.PNG">

### Theme

amro.omp.json

### What OS are you seeing the problem on?

Linux

### Which shell are you using?

bash

### Log output

```Shell
None
```

**Comment by JanDeDobbeleer** [comment]
@CoderLittleChen you need to configure for every user, yes.
